### DISCUSSION TITLE
Chat example: I can't attach files

### DISCUSSION THREAD

**Discussion by Helzio** [serverpod/serverpod#805] [Q&A]
When I try to attach a file in the chat I get this error from the server:

![Captura de pantalla 2023-04-04 a la(s) 13 49 48](https://user-images.githubusercontent.com/70969219/229904161-7292de92-57e8-4204-9c38-304d37ad85ce.png)

I'm trying to add different image format (like png, jpg, gif), but every time I get the same error.
Did I miss something or is an issue from the **_serverpod_chat_** library?

**Comment by Ifoegbu1** [comment]
Hello Helzio....I'm also trying to build a rich messaging chat using serverpod...Can we work together on that?
